File: c-sharp/7kyu/highest-and-lowest.cs
using System;
using System.Linq;

public static class Kata
{
  public static string HighAndLow(string numbers)
  {
    
    var numberList = numbers.Split(' ').Select( x => Convert.ToInt32(x) );
    
    return $"{numberList.Max()} {numberList.Min()}";
  }
  
}

File: javascript/7kyu/highest-and-lowest.js
function highAndLow(numbers){
  var high = numbers.split(' ').map( x => Number.parseInt(x) ).reduce( max, Number.MIN_SAFE_INTEGER);
  var low = numbers.split(' ').map( x => Number.parseInt(x) ).reduce( min, Number.MAX_SAFE_INTEGER );
  return high + ' ' + low;
}

function max( a, b ){
  return a > b ? a : b;
}

function min( a, b ){
  return a < b ? a : b;
}